{
    "errorHandling.deleteCredentials": "Unable to delete credentials.",
    "errorHandling.loadCredentials": "Unable to load credentials.",
    "credentials.missingEntryMessage.1": "Could not find an entry in the credential vault for the following:\n",
    "credentials.missingEntryMessage.2": "  Service = {0}\n",
    "credentials.missingEntryMessage.3": "  Account = {0}\n\n",
    "credentials.missingEntryMessage.4": "Possible Causes:\n",
    "credentials.missingEntryMessage.5": "  This could have been caused by any manual removal of credentials from your vault.\n\n",
    "credentials.missingEntryMessage.6": "Resolutions:\n",
    "credentials.missingEntryMessage.7": "  Recreate the credentials in the vault for the particular service in the vault."
}